### ISSUE TITLE
Gemfile's not analyzed

### ISSUE THREAD

**Issue by deivid-rodriguez** [houndci/hound#1321]
By default only `.rb` and `.rake` files are analyzed: https://github.com/houndci/hound/blob/master/app/models/linter/ruby.rb#L3, but default's `RuboCop` list of standard Ruby files is much bigger: https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L11-L29.

Ideally if a file is analyzed by a local run of `RuboCop`, it should also be analyzed by `Hound` when the file is changed in a PR. But even if this behaviour is not easily achievable, getting the default list in sync with `RuboCop`'s defaults would help.

**Comment by teoljungberg** [comment]
That's right @deivid-rodriguez - a PR would be most welcome to add it to our "whitelist".

**Comment by deivid-rodriguez** [comment]
Unfortunately I no longer use hound on any of the projects I work on, so I won't be able to provide a PR. Seems easy enough for anyone interested to pick up, though.
